app/src/main/java/com/example/sc2infoapp/TeamActivity.java

```diff
@@ -76,7 +77,8 @@ protected void onCreate(Bundle savedInstanceState) {
 
 
 
-        matches = new ArrayList<>();;
+        matches = new ArrayList<>();
+        externalMatches = new ArrayList<>();
```

Comment:
???

Reply:
Didn't touch TeamActivity, might've been from the merge I got from you last night

Reply:
Hm, by github history it seems that it's from base commetn commit